tests/virtual_machines_test.go

```diff
@@ -218,7 +218,7 @@ var _ = Describe("Virtual Machines", func() {
 			})
 		})
 		//2165 test postponed due to issue: https://github.com/k8snetworkplumbingwg/kubemacpool/issues/105
-		PContext("when trying to create a VM after a MAC address has just been released duo to a VM deletion", func() {
+		Context("when trying to create a VM after a MAC address has just been released duo to a VM deletion", func() {
```

Comment:
please keep the P on these tests as it is currently examined by @RamLavi  in another [task](https://issues.redhat.com/browse/CNV-4405)